docs/main/03-research/01-vendors/04-trello/04-trello.md

```diff
@@ -0,0 +1,198 @@
+  - **Members**: Here, you can invite people to your workspace via an invite link. You can also add **Guests**—people who are only granted access to specific boards, allowing them to view and edit those boards without joining the full workspace.
+
+  - **Settings**: This section lets you adjust workspace configurations. In the Free plan, options are limited; you can only set workspace visibility to either public or private and link your workspace to **Slack**.
+
+Finally, it’s worth noting that Trello allows you to create multiple workspaces, so you’re not limited to just one, making it easier to organize different projects or teams.
+
+## Board
+
+### Create Board
+
+On your workspace's home page, click on **Create new board**. Then some options will be provided for you:
+
+![](photos/06-create-board.jpg)
+
+- You can set a background (ether image or a gradient).
+
+- You can (you have to!) set a name to your board
+
+- You can set access to your board:
+- Change visibility:
+  - Private &rarr; Only board members can see this board.
+  - Workspace &rarr; All workspace members can view it.
+  - Public &rarr; Anyone on the internet can view it.
+
+- Finally, you can click on **Create** or you can **Start with a template** (which we discussed previously).
+
+### Inside the Board
+
+Welcome to your board’s page! Your workspace panel is still visible on the left, but you can close it if you prefer a cleaner view.
+
+![](photos/07-inside-the-board.jpg)
+
+We’ll explore this page from top to bottom, as it’s the most important part of Trello.
+
+#### 1. Workspace panel
+
+Here, we have the workspace panel again, with a few additional options:
+
+![](photos/08-workspace-panel.jpg)
+
+Section | Description
+:- | :-
+*More from Atlassian |  The icon on the far left (a 3x3 grid), which is primarily for advertising other Atlassian products.
+Search bar | Use this to find boards within your workspace.
+Notifications | See reminders, alerts, and other updates here.
+Information | This option leads to a Trello tour, which can be useful if you're new to Trello.
+Account | Access and modify your account settings here.
+
+#### 2. Board panel
+
+This section includes options specific to your board:
+
+![](photos/09-board-panel.jpg)
+
+Section | Description
+:- | :-
+Name | Double-click the board name to rename it.
+Star | Star your board to have it appear at the top of your boards list.
+Change visibility | Has already told.
+Customize views | The default view is the Kanban board, but Premium users can access additional views like Table, Calendar, Timeline, Dashboard, and Map. Only having the Kanban view can be a disadvantage if you’re looking for more layout options, which might make other tools appealing in comparison.
+Power-Ups | Add features and integrations like Google Drive or Slack to your board. Power-Ups work like extensions. E.g., using GitHub power-up, you can attach branches, commits, etc. directly to your cards.
+Automation | Will be explained later
```

Comment:
period

Reply:
I added period to all types of the sentences (even short ones)